packages/components/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Bug Fixes
 
+-   `ExternalLink`: Fix arrow direction for RTL languages. The external link arrow now correctly points to the top-left (↖) instead of top-right (↗) in RTL layouts. ([#73399](https://github.com/WordPress/gutenberg/pull/73399))
```

Comment:
```suggestion
-   `ExternalLink`: Fix arrow direction for RTL languages. The external link arrow now correctly points to the top-left (↖) instead of top-right (↗) in RTL layouts. ([#73400](https://github.com/WordPress/gutenberg/pull/73400))
```